### PR TITLE
Give more context to screenreader users on Account menu button.

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue:

- #6637

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/components/UserMenu/index.js [hunk 1]
```diff
@@ -162,6 +162,35 @@ export default function UserMenu() {
...
+			__( 'Google Account for %1$s', 'google-site-kit' ),
+			userFullName,
+			userEmail
+		);
+	}
+
+	if ( ! userFullName && userEmail ) {
+		accountLabel = sprintf(
+			/* translators: Account info text. 1: User's email address. */
+			__( 'Google Account (Email: %1$s', 'google-site-kit' ),
```
Comment (eugene-manuilov) [inline assets/js/components/UserMenu/index.js:188]:
```suggestion
			__( 'Google Account (Email: %1$s)', 'google-site-kit' ),
```

File: assets/js/components/UserMenu/index.js [hunk 2]
```diff
@@ -162,6 +162,35 @@ export default function UserMenu() {
...
+			userEmail
+		);
+	}
+
+	if ( userFullName && ! userEmail ) {
+		accountLabel = sprintf(
+			/* translators: Account info text. 1: User's (full) name. */
+			__( 'Google Account for %1$s', 'google-site-kit' ),
+			userFullName,
+			userEmail
```
Comment (eugene-manuilov) [inline assets/js/components/UserMenu/index.js:181]: I believe the `userEmail` variable is not needed here, right?

File: assets/js/components/UserMenu/index.js [hunk 3]
```diff
@@ -162,6 +162,35 @@ export default function UserMenu() {
...
+		);
+	}
+
+	if ( ! userFullName && userEmail ) {
+		accountLabel = sprintf(
+			/* translators: Account info text. 1: User's email address. */
+			__( 'Google Account (Email: %1$s', 'google-site-kit' ),
+			userFullName,
```
Comment (eugene-manuilov) [inline assets/js/components/UserMenu/index.js:189]: Same here, `userFullName` isn't needed right? This actually makes this translation incorrect.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]

File: assets/js/components/UserMenu/index.js (hunk 2 above)
Comment (tofumatt) [reply assets/js/components/UserMenu/index.js:181]: Oh, you're right, thanks. Removed! 👍🏻

File: assets/js/components/UserMenu/index.js (hunk 3 above)
Comment (tofumatt) [reply assets/js/components/UserMenu/index.js:189]: Argh, thanks 😅

**Pull request merged**